tests/is_balanced_or_bijected.py

```diff
@@ -0,0 +1,60 @@
+from supporting_components.graph_io import *
+from algorithms.color_refinement import color_refinement
+from algorithms.decide_gi import is_balanced_or_bijected
+
+
+# TODO: create a nice test that verifies the following:
```

Comment:
Deze TODO is nog niet weg aan het einde van je commits. Ik vind het goed om er testen bij te maken, maar zou mooi zijn dat het ook werkt ;) Ik heb ook net een test gemaakt, ook naar de stijl van Vera's testen. Misschien kun je daar even naar kijken? Dat zijn de pr voor de testfile van color_refinement en in de pr voor de copy van de graph

Reply:
ik move dat stuk code naar een ander issue